CHANGELOG.md

```diff
@@ -20,6 +87,8 @@ The first Python 3 ONLY capa version.
 - py3: drop Python 2 support #480 @Ana06
 - deps: bump ruamel yaml parser to 0.17.4 #519 @williballenthin
```

Comment:
Do we want to add dependencies updates to the CHANGELOG? We are missing smda, black and viv-utils updates.

If we want to add then, I think we should add a GitHub action for the dependabot PRs (to avoid that it is forgotten) and add the already missing updates. If not we should remove this entry.

I  prefer that we don't add them. It is easy that we miss something and there are easy ways to check the dependencies of a project

Reply:
btw, should it be `@dependabot` instead of `@williballenthin`? 😆

Reply:
I'd also say to NOT add them unless we can easily automate it.

Reply:
Continue discussion here: https://github.com/fireeye/capa/issues/550, I am going to merge this 😉